src/helpers/validateAngularVersion.js

```diff
@@ -0,0 +1,19 @@
+const { satisfies } = require('semver')
+
+// Ensure site is using Angular v17+.
+const validateAngularVersion = async function ({ failBuild, run }) {
+  const { stdout, exitCode } = await run('node', ['-p', "require('@angular/core/package.json').version"], {
```

Comment:
Why does this need to be done in a separate process? Could you just do an async import?

Reply:
I tried that for hours, but couldn't get it to work with the setup of the `demo` folder. Let's keep this in mind for a follow-up.

Reply:
Did you try using `require.resolve` with a paths option?

Reply:
that worked, thank you :D

Reply:
done in https://github.com/netlify/netlify-plugin-angular-universal/pull/67/commits/0e445a4b9747d9834a135a90c2f07bc026db808a